packages/bpk-component-image/src/withLazyLoading-test.js

```diff
@@ -40,18 +40,20 @@ describe('withLazyLoading', () => {
     const lazyLoadedImage = shallow(<LazyLoadedImage />, {
       disableLifecycleMethods: true,
     });
-    expect(lazyLoadedImage.first().is(MockImageComponent));
+    expect(lazyLoadedImage.children().type()).toBe(MockImageComponent);
```

Comment:
For these two tests, the linter was showing the error `No assertion was called on expect()  jest/valid-expect`.
I think these changes are testing what the previous ones intended to test, but better if you can review them 😅